### PR TITLE
Rename catalogue-item to work

### PR DESCRIPTION
**Pull request by davidpmccormick**
## Type
🚑 Health

## Value
Updating nomenclature to match platform team.

### All
- [x] Demoed to the relevant people
- [x] PR labelled and assigned
- [x] Any introduced code complexity has been flagged

### CODE REVIEW

**Comment by jamesgorrie** [comment]
Conflicts...

File: server/model/work.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// @flow
+
+export type Work = {|
+  id: string;
+  label: string;
+  descripition: string;
```
Comment (alexwlchan) [inline server/model/work.js]: descripition?

Comment (davidpmccormick) [inline server/model/work.js]: From the API response:
screen shot 2017-06-14 at 14 34 13

Comment (alexwlchan) [inline server/model/work.js]: I should have been clearer: the word "descripition" is misspelt. Should be "description".

File: server/model/work.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  creators: Array<string>;
+  type: string;
+  identifiers?: ?Array<{
+    source: string,
+    name: string,
+    value: string,
+    type: string
+  }>;
+|};
+
```
Comment (jamesgorrie) [inline server/model/work.js]: @kenoir thrift?

**Pull request merged**